### DISCUSSION TITLE
Trying to write custom time module

### DISCUSSION THREAD

**Discussion by Shouri14** [starship/starship#4947] [Q&A]
Trying to write a custom module, where it displays time when not in tmux.
When in tmux, the themes usually display time at the bottom, so no need for it in prompt

```
right_format = "${custom.time}"

[time]
disabled = false
use_12hr = true
format = '[ $time ]($style)'
time_format = '%I:%M %p'
utc_time_offset = '-5'

[custom.time]
disabled = false
style = "yellow bold"
format = '[ $time ]($style)'
when = '[[ -z "$TMUX" ]]'
```

Since, I can't add ```when``` to default ```time``` module, I thought of writing a custom module to achieve the same. 
I hoped to at least access ```$time``` from starship to use in ```custom.time``` but it doesn't print anything.

Any ideas or insight is appreciated, thanks.

**Comment by mickimnet** [comment]
A `[custom...]` module is basically a script run by a shell. To display the date, you need to add a line with something like `command = "date '+%I:%M %p'"` and replace `$time` in the `format` line with `$output`.
